glider_qc/glider_qc.py

```diff
@@ -434,17 +434,18 @@ def run_qc(config, ncfile, nc_path):
     file_name = nc_path.split('/')[-1]
 
     xyz = GliderQC(ncfile, config)
-    
+
     timedata = ncfile.variables['time']
     time_units = timedata.units
- 
+
     # Check the Time Array
     report = xyz.check_time(ncfile, nc_path)
 
     if len(report) == 0:
 
         # Check glider track coordinates
-        report = xyz.check_location(ncfile, report)
+        if 'qartod_location_flag' not in ncfile.variables:
```

Comment:
This was actually necessary to prevent lock contention and throwing an exception in a number of cases.